### PR TITLE
Update to work with jwt-go 3

### PR DESCRIPTION
**Pull request by srt32**
Per
https://github.com/dgrijalva/jwt-go/blob/master/MIGRATION_GUIDE.md#tokenclaims-is-now-an-interface-type
there has been an update to `Token.Claims` and prior to this update
there was the below error.

```
 $ go run main.go
 # command-line-arguments
 ./main.go:65: invalid operation: token.Claims["admin"] (type jwt.Claims
 does not support indexing)
 ./main.go:66: invalid operation: token.Claims["name"] (type jwt.Claims
 does not support indexing)
 ./main.go:67: invalid operation: token.Claims["exp"] (type jwt.Claims
 does not support indexing)
```

This PR updates the usage of `Token.Claims` to work.

It also runs `go fmt` and adjusted some formatting.

Thanks for the project!

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,86 +1,86 @@
...
-    token.Claims["name"] = "Ado Kukic"
-    token.Claims["exp"] = time.Now().Add(time.Hour * 72).Unix()
-    tokenString, err := token.SignedString(mySigningKey)
-    if(err != nil){
-    	log.Fatal(err)
-    }
-    w.Write([]byte(tokenString))
+var GetTokenHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	token := jwt.New(jwt.SigningMethodHS256)
+	claims := token.Claims.(jwt.MapClaims)
```
Comment (srt32) [inline main.go]: Here's the main change

**Comment by srt32** [comment]
Ping @kukicado.

**Comment by kukicado** [comment]
This is awesome. Thank you @srt32 . Sorry I didn't get a chance to review this earlier.

**Comment by srt32** [comment]
Thank you!

**Pull request merged**